cumulus/pallets/xcmp-queue/src/tests.rs

```diff
@@ -321,8 +321,9 @@ fn xcmp_queue_consumes_dest_and_msg_on_ok_validate() {
 	let message = Xcm(vec![Trap(5)]);
 
 	// XcmpQueue - check dest/msg is valid
-	let dest = (Parent, X1(Parachain(5555)));
-	let mut dest_wrapper = Some(dest.into());
+	let junctions: Junctions = [Parachain(5555)].into();
+	let dest = (Parent, junctions);
```

Comment:
Can this not be:

```suggestion
	let junctions: Junctions = [Parachain(5555)].into();
	let dest = (Parent, [Parachain(5555)]).into();
```

? If not, then it sounds like we should make conversion functions for it.

Reply:
Actually, we already have conversion functions for `(Parent, Parachain(5555))`, so no need for either the new variable or the array :smile:

Reply:
We might want to change that to `(Parent, [Parachain(5555)])`? WDYT?